src/sass/pages/offering/_value-props.scss

```diff
@@ -12,8 +12,8 @@
     padding-top: 200px;
   }
   @media(min-width: 1200px) {
-    padding-top: 192px;
-    padding-bottom: 100px;
+    padding-top: 164px;
+    padding-bottom: 64px;
```

Comment:
Again, another weird artifact I'm seeing here is that the bottom padding seems to overlap the next section, meaning the whitespace here is likely less than 64px.

![Screen Shot 2022-12-06 at 12 49 13 PM](https://user-images.githubusercontent.com/5032149/206008583-81bedadc-10f6-4eb2-a3ea-9cc46ff972b6.png)

Reply:
Adjusted the container below with tabs so padding is correct.